osmosis/assetlist.json

```diff
@@ -433,6 +433,183 @@
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/yieldeth.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/yieldeth.svg"
+      },
+      "keywords": ["osmosis-main"]
+    },
+    {
+      "description": "The native staking token of dYdX Protocol.",
+      "denom_units": [
+        {
+          "denom": "ibc/831F0B1BBB1D08A2B75311892876D71565478C532967545476DF4C2D7492E48C",
+          "exponent": 0,
+          "aliases": [
+            "adydx"
+          ]
+        },
+        {
+          "denom": "dydx",
+          "exponent": 18
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/831F0B1BBB1D08A2B75311892876D71565478C532967545476DF4C2D7492E48C",
+      "name": "dYdX",
+      "display": "dydx",
+      "symbol": "DYDX",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "dydx",
+            "base_denom": "adydx",
+            "channel_id": "channel-3"
+          },
+          "chain": {
+            "channel_id": "channel-6787",
+            "path": "transfer/channel-6787/adydx"
+          }
+        }
+      ],
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/dydx/images/dydx-circle.svg"
+        }
+      ],
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/dydx/images/dydx-circle.svg"
+      },
+      "coingecko_id": "dydx",
+      "keywords": [
+        "osmosis-main"
+      ]
+    },
+    {
+      "description": "The native token of Quasar",
+      "denom_units": [
+        {
+          "denom": "ibc/1B708808D372E959CD4839C594960309283424C775F4A038AAEBE7F83A988477",
+          "exponent": 0
+        },
+        {
+          "denom": "uqsr",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/1B708808D372E959CD4839C594960309283424C775F4A038AAEBE7F83A988477",
+      "name": "Quasar",
+      "display": "qsr",
+      "symbol": "QSR",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "quasar",
+            "base_denom": "uqsr",
+            "channel_id": "channel-1"
+          },
+          "chain": {
+            "channel_id": "channel-688",
+            "path": "transfer/channel-688"
+          }
+        }
+      ],
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/quasar/images/quasar.png"
+        }
+      ]
+    },
+    {
+      "description": "",
+      "denom_units": [
+        {
+          "denom": "uqsr",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "qsr",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "type_asset": "sdk.coin",
+      "base": "uqsr",
```

Comment:
This asset does no exist (in this form) on Osmosis chain

Reply:
You are right, this is wrong but also is redundant as the previous object is also about $QSR. See here: https://github.com/cosmos/chain-registry/pull/3240/files/4bfc09a6600d8d9740b42887159044d3f3c62097#diff-afc1495905a465ffe02b6fa4b659d5aabd602f9eb8c9276b06125b3591fc98a1R562

I removed the whole redundant block. It should be fine now. Thanks for reviewing this.